client/src/pages/EmailNeedsConfirmation/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {CssBaseline, Grid, Paper, Typography} from '@material-ui/core'
+import HighlightOffOutlinedIcon from '@material-ui/icons/HighlightOffOutlined';
+
+import {Link} from 'react-router-dom'
+
+import useStyles from './EmailNeedsConfirmationStyles'
+
+const EmailNeedsConfirmation = props => {
```

Comment:
What component will pass props to EmailNeedsConfirmation?

Reply:
Valid point. Instead of waiting props I've implemented link onClick function:
    const goBack = e => {
        e.preventDefault();
        history.goBack();
    }